spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -545,6 +545,8 @@
       user: user
     )
 
+    user.update!(webauthn_platform_recommended_dismissed_at: Time.zone.now)
```

Comment:
I think we should prefer to setting `webauthn_platform_recommended_dismissed_at` in the factory rather than have to update like this

Reply:
Something like:

```diff
     # spec/factories/users.rb

     created_at { Time.zone.now }
+    webauthn_platform_recommended_dismissed_at { Time.zone.now }
```

Reply:
Discussed a bit offline, I'm not sure that this suggestion is the right way to go, so I'm resolving this.